### PR TITLE
fix: Lottery user ticket amount info text not translated correctly

### PR DESCRIPTION
**Pull request by memoyil**
To review:

https://deploy-preview-2059--pancakeswap-dev.netlify.app/

To reproduce: 

1. Go to lottery
2. Connect your account
2. Check next draw section
3. Change language
4. See Your tickets part not translated correctly

### CODE REVIEW

File: src/views/Lottery/components/NextDrawCard.tsx [hunk 1]
```diff
@@ -127,6 +127,16 @@ const NextDrawCard = () => {
     return ''
   }
 
+  let ticketRoundText
+  if (userTicketCount > 1) {
+    ticketRoundText = t('You have %amount% tickets this round', { amount: userTicketCount })
+  } else {
+    ticketRoundText = t('You have %amount% ticket this round', { amount: userTicketCount })
+  }
```
Comment (hachiojidev) [inline src/views/Lottery/components/NextDrawCard.tsx:135]: We can use a ternary here instead.
`const ticketRoundText = userTicketCount > 1 ?...`

File: src/views/Lottery/components/NextDrawCard.tsx [hunk 2]
```diff
@@ -127,6 +127,16 @@ const NextDrawCard = () => {
...
+  const ticketRoundSplit = ticketRoundText.split(userTicketCount.toString())
+  const youHaveText = ticketRoundSplit[0]
+  const ticketsThisRoundText = ticketRoundSplit[1]
```
Comment (hachiojidev) [inline src/views/Lottery/components/NextDrawCard.tsx:138]:
```
const [youHaveText, ticketsThisRoundText] = ticketRoundText.split(userTicketCount.toString())
```

File: src/views/Lottery/components/PreviousRoundCard/Body.tsx [hunk 3]
```diff
@@ -70,6 +70,17 @@ const PreviousRoundCardBody: React.FC<{ lotteryNodeData: LotteryRound; lotteryId
...
+  const totalTicketNumber = userDataForRound ? userDataForRound.totalTickets : 0
+  if (totalTicketNumber > 1) {
+    ticketRoundText = t('You had %amount% tickets this round', { amount: totalTicketNumber })
+  } else {
+    ticketRoundText = t('You had %amount% ticket this round', { amount: totalTicketNumber })
+  }
+  const ticketRoundSplit = ticketRoundText.split(totalTicketNumber.toString())
+  const youHadText = ticketRoundSplit[0]
+  const ticketsThisRoundText = ticketRoundSplit[1]
+
```
Comment (hachiojidev) [inline src/views/Lottery/components/PreviousRoundCard/Body.tsx:83]: Same as above.

**Review by hachiojidev — CHANGES_REQUESTED** [review]
@ChefHutch Do you want to take a look at this as well?

File: src/views/Lottery/components/NextDrawCard.tsx (hunk 1 above)
Comment (memoyil) [reply src/views/Lottery/components/NextDrawCard.tsx]: Done

File: src/views/Lottery/components/NextDrawCard.tsx (hunk 2 above)
Comment (memoyil) [reply src/views/Lottery/components/NextDrawCard.tsx]: Done

File: src/views/Lottery/components/PreviousRoundCard/Body.tsx (hunk 3 above)
Comment (memoyil) [reply src/views/Lottery/components/PreviousRoundCard/Body.tsx]: Done

**Review by ChefHutch — APPROVED** [review]
👍  ty memoyil

**Review by hachiojidev — APPROVED** [review]

**Pull request merged**